### PR TITLE
add simplified VS code instructions

### PR DESCRIPTION
**Pull request by gebhardtr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,12 @@ podman compose -f tools/docker-compose/compose.yaml --env-file ./.env up
 
 ### Configure VS Code to connect to your local machine
 
-TODO
+1. Navigate to `Settings` (Code -> Settings -> Settings)
```
Comment (hasys) [inline README.md:43]: This is Mac specific. Should be
```
File > Preferences (Code > Preferences or Code > Settings on macOS) > Settings
```

**Review by hasys — APPROVED** [review]
Hi @gebhardtr looks good to me, just one optional improvement

**Review by ganeshrn — APPROVED** [review]

**Comment by manstis** [comment]
@hasys Could you complete this PR now Richard has left us?

I agree with your comment but otherwise the PR looks good to me (but we're not going to get Richard to update it).

**Comment by manstis** [comment]
IDK whether it's worth trying to rebase this PR?

I re-triggered the Konflux step but it failed again:
```
Pipeline ansible-lightspeed-tenant/ansible-ai-connect-service-on-pr-1152-62xjq can't be Run; it contains Tasks that don't exist: Couldn't retrieve Task "resolver type bundles\nname = clamav-scan\n": invalid runtime object: strict decoding error: unknown field "spec.volumes[0].emptydir", unknown field "spec.volumes[1].emptydir"
```
Otherwise we may need @SiddharthRajaraman to comment.

<sigh> What should have been a simple change.....

**Comment by hasys** [comment]
Hi @manstis, sorry my VS Code did some auto formatting and broke everything. Now fixed and minimal change applied. Also cherry-picked to the main branch without any issues. Ready for review 👍

**Review by manstis — APPROVED** [review]
LGTM :+1: 

Thank-you for continuing with this PR.

**Pull request merged**